### DISCUSSION TITLE
Weird transparency bug

### DISCUSSION THREAD

**Discussion by rockhaven202** [Vapok/JotunnBackpacks#8] [General]
Not sure if this is the right place to post but I'm getting a strange transparency bug with both backpacks. If you look closely you can see my armor, and workbench upgrades through the backpacks.
![20230108144747_1](https://user-images.githubusercontent.com/50389065/211218488-2054cf78-9343-4c51-bd90-c762c4800b38.jpg)
![20230108144754_1](https://user-images.githubusercontent.com/50389065/211218490-e2451a0a-1d44-4325-89bc-400d12744ecf.jpg)
![20230108144813_1](https://user-images.githubusercontent.com/50389065/211218491-39a9ff18-1839-4e23-b9b2-2841f876cdbd.jpg)
![20230108145003_1](https://user-images.githubusercontent.com/50389065/211218493-c5a62486-3eb8-4e68-9cc1-a54a54694ccc.jpg)
![20230108145022_1](https://user-images.githubusercontent.com/50389065/211218494-d742f165-7ff9-4b89-8580-228c6f6313f7.jpg)

**Comment by Vapok** [comment]
I believe this is due to the issue that the original mod wasn't Vulcan supported. 

I am working on converting the Assets to be available on Vulcan.

**Comment by rockhaven202** [comment]
I still get the transparency not using Vulcan, is that because of the same
issue?
